drivers/sensor/bme680/bme680.h

```diff
@@ -29,19 +74,22 @@
 #define BME680_REG_CTRL_HUM             0x72
 #define BME680_REG_CTRL_MEAS            0x74
 #define BME680_REG_CONFIG               0x75
-#define BME680_REG_MEM_PAGE             0xf3
+#define BME680_REG_MEM_PAGE             0x73
```

Comment:
There are several changes in this commit that are unrelated to adding support for the SPI interface. For the most part, they are minor enough that I don't want to quibble over them. But changes like this really shouldn't be squashed into a commit that claims to be only adding support for a new bus type. Please split this into its own commit with a separate justification in the commit log, so it can be reviewed separately. If the register value is simply wrong, it's a separate bug fix patch.

(Ideally, if you can take the time to split other unrelated changes (like moving code around, adding log messages, etc.) into a separate 'prep work' commit as well, that would make this whole patch easier to review, but as I said, I otherwise wouldn't have quibbled this time.)

Reply:
The register value is not *really* wrong. Confusingly it just had the write bit set already.
Regardless, your criticism is well justified and I will take the time to split up the unrelated changes into their own commits, get rid of the log messages and add constants for the SPI read/write masks :+1:

Reply:
Thank you for doing this